ReadMe.md

```diff
@@ -4,3 +4,5 @@ Cats are better in VR.
 
 ## Prerequisites
 - [Unity](https://unity3d.com/get-unity)
+
+cupps was here and cats are better in vr
```

Comment:
This is garbage what is this? "cupps" is a proper noun and should be capitalized. Also we already stated that cats are better in VR, why restate it?